src/gui/viewer.hpp

```diff
@@ -126,6 +129,11 @@ namespace vtfview
 		void set_pixmap(const QImage& pixmap);
 		void set_vtf(VTFLib::CVTFFile* file);
 
+		// Should probably make the checkerboard size dependent on viewer size...
+		const int checkerboard_size = 512;
+		//const int checkerboard_quality = 16;
```

Comment:
Well in the meantime, let's make these compile time constants with `static constexpr inline` instead of `const` (`inline` is technically optional, `constexpr` should add it, but I find it clearer. up to you)